package.json

```diff
@@ -46,9 +54,10 @@
     "clean": "gts clean",
     "compile": "tsc",
     "fix": "gts fix",
-    "posttest": "npm run check",
-    "start": "node build/server/server.js",
-    "dev": "ts-node-dev src/server/server.ts",
+    "test": "jest ${1} --forceExit",
```

Comment:
To be sure: do we need the `forceExit` flag? It could be an indication that the tear down in the tests isn't working properly.

Reply:
Unfortunately, we do. Otherwise we run into:

```
Jest did not exit one second after the test run has completed.

This usually means that there are asynchronous operations that weren't stopped in your tests. Consider running Jest with `--detectOpenHandles` to troubleshoot this issue.
```

The only test case that throws this error is the one that tests our query timeout functionality. This is probably caused by https://github.com/comunica/comunica/issues/70.